src/modules/menu-focus.js

```diff
@@ -11,6 +11,7 @@ export function getMenuElements() {
 			type: 'menu',
 			handler: makeHandler( menu.location ),
 			title: 'menu',
+			onPartialUpdate: onPartialUpdate,
```

Comment:
Super minor, but since this is ES6 I think we can just make this line `onPartialUpdate,`